docs/docs/explore/mcp.md

```diff
@@ -65,9 +65,52 @@ This ensures **trustworthy, governed analytics** while empowering users to **sel
 
 To use the Rill MCP server, you'll need:
 
-- An **MCP client** (we recommend [Claude Desktop](https://claude.ai/download), but you can use any compatible client. [Why?](#configure-claude-desktop))
+- An **MCP client** (we recommend [Claude Desktop](https://claude.ai/download), but you can use any compatible client. [Why?](#edit-claude-desktop-configuration))
 - A **running Rill project** (locally or hosted on Rill Cloud)
-- **Node.js**, which can be downloaded from [nodejs.org](https://nodejs.org/en)
+- **Node.js**, which can be downloaded from [nodejs.org](https://nodejs.org/en) (not required for OAuth connection method)
```

Comment:
I think we should remove this requirement since most people won't need it anymore, and it makes MCP look more complex.

Maybe we can move the requirement to the alternative section, and say something like "Note: If you select this option, you must have Node.js installed on your system.".